linera-core/src/client/mod.rs

```diff
@@ -1850,7 +1822,7 @@ where
         if missing_blobs.is_empty() {
             Ok(())
         } else {
-            Err(ChainClientError::BlobsNotFound(missing_blobs))
+            Err(NodeError::BlobsNotFound(missing_blobs).into())
```

Comment:
How can you tell this is a _remote_ node error? 🤔

Reply:
AFAIU we use `NodeError` to errors related to the validators 🤔
We're trying to find any validator here that knows that some blobs exist. If no validator has knowledge a blob, then that blob is missing from the validators. Then it should be a `NodeError`, no? At least that's my understanding of the semantics of `NodeError` that we're trying to have

Reply:
Right, in this particular function it makes sense to return `Remote*` as we queried the `remote_node` above.

FYI, `receive_certificate_for_blob` (singular) is never used.

Reply:
Yes, we should probably rename `NodeError` to `RemoteError` and also reorganize it. (https://github.com/linera-io/linera-protocol/issues/2857)